app/components/common/RatingSlice.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import { View, StyleSheet } from 'react-native'
+import RNText from './RNText'
+import colors from '../../config/colors'
+
+const RatingSlice = ({ title, redValue, amberValue, greenValue }) => {
+  return (
+    <View style={styles.container}>
+      <View style={styles.titleContainer}>
+        <RNText style={styles.title}>{title}</RNText>
+      </View>
+      <View style={styles.row}>
+        <View style={styles.columnHeader}>
+          <RNText style={styles.redTitle}>Red</RNText>
+        </View>
```

Comment:
You could look a splitting this into its own component. Would clean up this file by removing a few duplicate lines. For the style I would pass in the colour from the redTitle etc. But have a default that is aways applies to handle setting the font size.